pbuilder/pbuilderrc

```diff
@@ -5,9 +5,6 @@
 #
 # read pbuilderrc.5 document for notes on specific options.
 
-DEB_BUILD_OPTIONS="parallel=$(nproc) nocheck"
-export DEB_BUILD_OPTIONS="parallel=$(nproc) nocheck"
```

Comment:
We need parallel=$(nproc) this makes sure we build in parallel aka -j$(nproc) 

This should also get overwritten by https://github.com/ubports/build-tools/blob/master/build-binary.sh#L22 bit it might be the other way around :/

Reply:
Oh! sorry little quick on the trigger there, this removes it all together so the one in build-binary is the one that get used!

Reply:
So this does make sense!